sdk/eventhub/Azure.ResourceManager.EventHubs/README.md

```diff
@@ -1,69 +1,51 @@
-
-### Prerequisites
-
-* You must have an [Azure subscription](https://azure.microsoft.com/free/)
+Install the Azure Management EventHubs client library for .NET with [NuGet](https://www.nuget.org/):
 
-### Authenticate the client
+```PowerShell
+Install-Package Azure.ResourceManager.EventHubs
+```
 
-If your library requires authentication for use, such as for Azure services, include instructions and example code needed for initializing and authenticating.
+## Documentation
 
-For example, include details on obtaining an account key and endpoint URI, setting environment variables for each, and initializing the client object.
+Various documentation is available to help you learn how to use this package
 
-## Key concepts
+- [Quickstart](https://github.com/Azure/azure-sdk-for-net/blob/master/doc/mgmt_preview_quickstart.md)
+- [API References](https://docs.microsoft.com/dotnet/api/?view=azure-dotnet)
+- [Authentication](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/identity/Azure.Identity/README.md)
 
-The *Key concepts* section should describe the functionality of the main classes. Point out the most important and useful classes in the package (with links to their reference pages) and explain how those classes work together. Feel free to use bulleted lists, tables, code blocks, or even diagrams for clarity.
+## Need help?
 
-## Examples
-
-Include code snippets and short descriptions for each task you listed in the [Introduction](#introduction) (the bulleted list). Briefly explain each operation, but include enough clarity to explain complex or otherwise tricky operations.
-
-## Troubleshooting
-
-Describe common errors and exceptions, how to "unpack" them if necessary, and include guidance for graceful handling and recovery.
-
-Provide information to help developers avoid throttling or other service-enforced errors they might encounter. For example, provide guidance and examples for using retry or connection policies in the API.
-
-If the package or a related package supports it, include tips for logging or enabling instrumentation to help them debug their code.
-
-## Next steps
-
-* Provide a link to additional code examples, ideally to those sitting alongside the README in the package's `/samples` directory.
-* If appropriate, point users to other packages that might be useful.
-* If you think there's a good chance that developers might stumble across your package in error (because they're searching for specific functionality and mistakenly think the package provides that functionality), point them to the packages they might be looking for.
+-   File an issue via [Github
+    Issues](https://github.com/Azure/azure-sdk-for-net/issues)
+-   Check [previous
+    questions](https://stackoverflow.com/questions/tagged/azure+.net)
+    or ask new ones on StackOverflow using azure and .NET tags.
```

Comment:
nit:  "... using the `azure` and `.net` tags."   _(Stack Overflow tags are lower case)_